mutiny-wasm/src/error.rs

```diff
@@ -28,7 +27,7 @@ pub enum MutinyJsError {
     ConnectionFailed,
     /// The invoice or address is on a different network
     #[error("The invoice or address is on a different network.")]
-    IncorrectNetwork(Network),
+    IncorrectNetwork,
```

Comment:
Need to make sure the frontend isn't trying to use this anywhere

Reply:
The frontend just uses the strings so it wasn't